app/shared/inputs/ImageInput.js

```diff
@@ -7,6 +7,7 @@ export default function ImageInput({
   imageUrl,
   imageLabel = 'Image',
   onChange,
+  maxSize = 1 * 1024 * 1024,
```

Comment:
I don't know about correctness, just want to suggest confirming alignment on max size. 

There's this default value (1 * 1024 * 1024), there's the input value for this component in HeroStep.js (0.5 * 1024 * 1024), there's the default value for the corresponding parameter in FileDropZone.js (5000000), and then we make an image size recommendation to the user of "1280x640px. PNG or JPG format." That recommendation should probably say there's a file size limit of 0.5 MB. 

Maybe all of those numbers are correct as they are, just wanted to call out for verification.